src/components/AssignedMembers/AssignedMembersContainer.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Headline,
+  Badge,
+  Loading,
+  Accordion,
+} from '@folio/stripes/components';
+
+import useAssignedUsers from './hooks/useAssignedUsers';
+import AssignedMembersList from './AssignedMembersList';
+
+const AssignedMembersContainer = ({ permissionsSet, expanded, onToggle }) => {
```

Comment:
> Since we need to reuse this component in the consortium manager, we need the ability to explicitly specify the tenant for which we are going to load. It seems we need a `tenantId` property that needs to be passed as the value of `X-Okapi-Tenant`.